content/concepts/did-ddo.md

```diff
@@ -269,6 +285,14 @@ Example:
       "datatokenAddress": "0x124",
       "serviceEndpoint": "https://myprovider.com",
       "timeout": 0,
+      "resources":{
```

Comment:
if used like this also needs to be added to full DDO example at the end of document. Also proper syntax formatting please as the other code, e.g. space after `:`, same indentation and so on. Ideally just install Prettier in your editor so happens on save also for code snippets in markdown.

Reply:
done